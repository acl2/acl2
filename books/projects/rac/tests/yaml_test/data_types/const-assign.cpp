// RAC begin

int foo() {
  const int a = 1;
  a = 3;
  return a;
}
