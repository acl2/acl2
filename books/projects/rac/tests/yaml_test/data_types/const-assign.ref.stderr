const-assign.cpp:5 (2-3): Assignment of read-only variable a
  5 |   a = 3;
    |   ^
note: defined here
  4 |   const int a = 1;
    |             ^
